hardware/HardwareMonitor.h

```diff
@@ -96,6 +96,9 @@ class CHardwareMonitor : public CDomoticzHardwareBase
 	float GetProcessMemUsage();
 	float GetMemUsageLinux();
 #endif
+#if defined (__FreeBSD__)
+	float GetMemUsageLinux();
+#endif
 #if defined (__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
@xbeaudouin , how does this relate to this OR ? We compile 2 memory function for FreeBSD (GetMemUsageLinux and GetMemUsgaeOpenBSD)?

Reply:
Well the original compilation looks for GetMemUsageLinux() as you see in the following error while compiling.
```
hardware/HardwareMonitor.cpp:743:26: error: out-of-line definition of 'GetMemUsageLinux' does not match any declaration in 'CHardwareMonitor'
        float CHardwareMonitor::GetMemUsageLinux()
                                ^~~~~~~~~~~~~~~~
1 warning and 1 error generated.
```

Aslo on this code there is also specific FreeBSD stuff. So FreeBSD seems to need this function.

Reply:
@xbeaudouin ok clear. Maybe than it is more clear then to make it something like
`#if defined (__Linux__) || defined (__FreeBSD)`
and remove the function declaration 'float GetMemUsageLinux()' from the lines above (97)

This way it is only referenced once in the header file and it is visible at once for which systems it is needed.